lib/Threading/Internal/Thread.php

```diff
@@ -53,44 +53,44 @@ public function run() {
          * thread-safe objects.
          */
         $paths = [
-            \dirname(__DIR__, 5) . \DIRECTORY_SEPARATOR . 'autoload.php',
-            \dirname(__DIR__, 3) . \DIRECTORY_SEPARATOR . 'vendor' . \DIRECTORY_SEPARATOR . 'autoload.php',
+            \dirname(__DIR__, 3) . '/vendor/autoload.php',
+            \dirname(__DIR__, 5) . '/autoload.php',
```

Comment:
@kelunik I then propose another approach: look up in get_included_files() what autoloader has been used and include that. (Do this in constructor, before the new Thread is started)